example.py

```diff
@@ -0,0 +1,25 @@
+import json
```

Comment:
Let's put this file in a directory named `examples` and rename it something like `revertrisk_examples.py`